crates/turborepo-lib/src/query/file.rs

```diff
@@ -18,26 +23,119 @@ impl File {
     }
 }
 
+#[derive(Clone, Debug, SimpleObject)]
+struct All {
+    count: usize,
+}
+
+#[derive(Union)]
+enum PackageMapping {
```

Comment:
Union types in GraphQL are pretty annoying. Maybe I should change this to just return all the packages? I'm a little worried that could be too much data.